content/en/blog/_posts/2024-10-28-memory-manager-moves-to-ga/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: blog
+title: "Memory Manager goes GA"
+date: 2024-10-28
+slug: memory-manager-goes-ga
+author: >
+  [Talor Itzhak](https://github.com/Tal-or) (Red Hat)
+---
+
+With Kubernetes 1.32, the Memory Manager has officially graduated to General Availability (GA),
+marking a significant milestone in the journey toward efficient and predictable memory allocation for containerized applications.
+Since Kubernetes v1.22, where it graduated to [Beta](https://kubernetes.io/blog/2021/08/11/kubernetes-1-22-feature-memory-manager-moves-to-beta/)
+the Memory Manager proved itself reliable, stable and a good complementary feature for the [CPU Manager](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/).
+The Memory Manager enables the user to allocate exclusive
+memory for `Guaranteed` [QoS class](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#qos-classes) pods.
+Most of the changes introduced since the Beta are bug fixes, internal refactoring and 
+observability improvements, such as metrics and better logging.
```

Comment:
since long time passed since the beta graduation, it can be useful to add a quick recap of what memory manager does and how it integrates in the kubelet, deferring to the existing docs for more details.